.gitlab-ci.yml

```diff
@@ -365,6 +365,8 @@ build_win:
   before_script: []
   script:
     - call .gitlab-scripts/setup_artifactory.cmd
+    - set "PATH=C:\tools\msys64\mingw64\bin;%PATH%"
```

Comment:
That path comes after enabling ridk development chain.
If you have to set this there after enabling ridk, smth messed the env